lib/auth/userloginstate/generator_test.go

```diff
@@ -258,13 +259,32 @@ func newAccessList(t *testing.T, name string, members []string, roles []string,
 			Roles:  roles,
 			Traits: traits,
 		},
-		Members: alMembers,
 	})
 	require.NoError(t, err)
 
 	return accessList
 }
 
+func newAccessListMembers(t *testing.T, accessList string, members ...string) []*accesslist.AccessListMember {
+	alMembers := make([]*accesslist.AccessListMember, len(members))
+	for i, member := range members {
+		var err error
+		alMembers[i], err = accesslist.NewAccessListMember(header.Metadata{
+			Name: member,
+		}, accesslist.AccessListMemberSpec{
+			AccessList: accessList,
+			Name:       member,
+			Joined:     time.Now(),
```

Comment:
Shouldn't a fake clock be used here?

Reply:
The way these tests are designed it doesn't really matter, but may as well.